src/uk/ac/open/audio/streaming/PlayerProgress.java

```diff
@@ -547,21 +554,35 @@ else if(currentState==State.PLAYING || currentState==State.RECORDING)
 			buffer2.setColor(timeFG);
 			buffer2.setFont(TIMEFONT);
 
-			int ms=playMilliseconds;
-			String minutes=ms/60000+"";
-			if(minutes.length()<2) minutes="0"+minutes;
-			ms%=60000;
-			String seconds=ms/1000+"";
-			if(seconds.length()<2) seconds="0"+seconds;
-			ms%=1000;
-			String tenths=ms/100+"";
-			String time=minutes+":"+seconds+"."+tenths;
-			buffer2.drawString(time, TEXTLEFTMARGIN, TEXTTOPMARGIN);
+			int ms;
+			if (TabApplet.pauseFlag == false) {
```

Comment:
Trivia: brace positioning doesn't match coding style (the { should be on its own line)